simplq/src/store/appSlice.js

```diff
@@ -10,6 +10,7 @@ const appSlice = createSlice({
     currentJoinerStep: window.localStorage.getItem('currentJoinerStep'),
     aheadCount: null,
     errorText: '',
+    notificationPermission: Notification.permission,
```

Comment:
see comment below, if it makes sense, then this would be:

```
notificationPermission: window.localStorage.getItem('notificationPermission'),
```

Reply:
window.localStorage.getItem('notificationPermission' ) || Notification.permission

Reply:
see comment below. lets store to local storage only if there is a need to.